### DISCUSSION TITLE
Elements through which a fault passes

### DISCUSSION THREAD

**Discussion by anneschulz1991** [gempy-project/gempy#737] [Q&A]
Hello all, I know that in gempy there is a command that specifies all the cells of the grid through which a fault passes. Unfortunately I have forgotten what this command is. Can someone help me?  Many greetings Anne

**Comment by Japhiolite** [comment]
Hi Anne, 

I think you mean the method showed [here](https://github.com/cgre-aachen/gempy/issues/713#issuecomment-1217661946) in #713 

Best
Jan
